File: Database/src/main/java/sep3/database/Model/MessageList.java
package sep3.database.Model;

import java.util.ArrayList;

public class MessageList {
    private ArrayList<Message> messages;
    public MessageList()
    {
        this.messages = new ArrayList<>();

    }

    public MessageList(ArrayList<Message> messages) {
        this.messages = messages;
    }
    public void addMessage(Message message)
    {
        messages.add(message);
    }
    public void removeMessage(Message message)
    {
        messages.remove(message);
    }
    public ArrayList<Message> getMessages(){
        return messages;
    }
}

File: Application/Application/Model/MessageList.cs
using System.Collections;
using System.Collections.Generic;
using System.Threading.Tasks;
using Tier2.Model;

namespace Application.Model
{
    public class MessageList
    {
        public List<Message> Messages { get; set; }

        public MessageList()
        {
            this.Messages = new List<Message>();
        }

        public MessageList(List<Message> messages)
        {
            this.Messages = messages;
        }

        public void addMessage(Message message)
        {
            Messages.Add(message);
        }

        public void removeMessage(Message message)
        {
            //we can do with message ID as well
            Messages.Remove(message);
        }
    }
}